lesson1.js

```diff
@@ -0,0 +1,113 @@
+//Task 1
+
+const loop = (times=0, callback=null) => {
+    if (callback) {
```

Comment:
```suggestion
    if (typeof callback === 'function') {
```
проверяем функция ли это:)